packages/bundle-size/src/index.js

```diff
@@ -1,13 +1,14 @@
 const yargs = require('yargs');
 
 const cliSetup = yargs
-  .commandDir('commands')
+  .commandDir('commands', { exclude: /.test.js$/ })
```

Comment:
`commandDir` was evaluation also test files ? 👀. 
- is that regex ok ? shouldn't it be `/\.test\.js$/`

Reply:
> commandDir was evaluation also test files ? 👀.

Yes

```
yarn run v1.22.10
$ office-ui-fabric-react\node_modules\.bin\bundle-size --help
office-ui-fabric-react\packages\bundle-size\src\commands\compareReports.test.js:1
const getRemoteReport = jest.fn();
                        ^

ReferenceError: jest is not defined
    at Object.<anonymous> (office-ui-fabric-react\packages\bundle-size\src\commands\compareReports.test.js:1:25)
    at Module._compile (internal/modules/cjs/loader.js:1015:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1035:10)
```

> is that regex ok ? shouldn't it be /\.test\.js$/

You're right, dot should be escaped 👍